snipssonos/use_cases/inject_entities.py

```diff
@@ -0,0 +1,23 @@
+import logging
+from snipssonos.shared.use_case import UseCase
+from snipssonos.shared.response_object import ResponseSuccess, ResponseFailure
+
+
+class InjectEntitiesUseCase(UseCase):
+
+    def __init__(self, music_custom_service, inject_entities_service):
+        self.music_custom_service = music_custom_service
```

Comment:
I would rename `music_custom_service` to something more explicit

Reply:
Do you mean what you already suggested for SpotifyCustomService to SpotifyCustomizationService? In other words, call this `music_customization_service`? Please give me a an example otherwise.

Reply:
Let's call it `music_customization_service` ;) !